resources/assets/less/bem/mp-history-event.less

```diff
@@ -0,0 +1,79 @@
+/**
+*    Copyright 2016 ppy Pty. Ltd.
```

Comment:
some of the license headers (like this one) are not properly aligned.

(don't bother fixing them all, just the ones you added here)